### ISSUE TITLE
fix: accessibility issues

### ISSUE THREAD

**Issue by lovelacecoding** [WalkingPal/walking-pal-web#30]
The website has some accessibility issues like low color contrast and missing alt text for images. These should be fixed so that the website does better on Google, and to help people with eye-sight issues (accessibility).

See the issues yourself, you can look at [WAVE analysis of WalkingPal](https://wave.webaim.org/report#/https://walkingpal.in/)

-------------
In **Home page** would like to fix the following things:
* Color contrast in navbar,  
* Color contrast in 'Get your walking buddy' section
* Color contrast in 'Reimagine your daily commute' section
* Alternative text for images
* Empty links

--------------

----------------
### Progress

- [x] Home page (@lovelacecoding 
- [ ] About page
- [ ] Company page
- [ ] FAQs page
- [ ] Register page

**Comment by ashuvssut** [comment]
please make separate PRs for different pages

**Comment by lovelacecoding** [comment]
Sure thing!

**Comment by astro3012** [comment]
@ashuvssut The questions and answers in the FAQ could also use a bit of change in contrast/weights to establish a better visual hierarchy and make it easier to differentiate. As this similar issue was already up, adding this suggestion as a comment.

**Comment by ashuvssut** [comment]
@astro3012 right!

When I was creating those sections we followed this [figma design board](https://www.figma.com/file/seUqjLLCz2X22Ymm6PZBe6/Website-%26-Other-Components?node-id=901%3A2897) created by the team

I really do not understand what contrast changes are... I guess they might be simple color changes only
make sure we do not deviate from the original color theme too much. tbh I suck at design skills. I am just good at replicating designs in code. @lovelacecoding can you just show me what kind of contrast changes were you referring to (using screenshots)

I am completely fine with ARIA labels n stuff. I am just little concerned about contrast changes. A few screenshots showing how it looked"before" and how its gonna look "after" the contrast change will help.

**Comment by lovelacecoding** [comment]
That you already have a color palette makes it way easier @ashuvssut. Let me change it more to that. I'll come back with screenshots shortly

**Comment by ashuvssut** [comment]
Oh no, don't refer the color palette section given in the figma design. @lovelacecoding The color palette we follow is in theme.ts. We created that color palette by refering the designs in figma. we have `.docs/style-guide.md` that  describes the palette. Refer theme.ts.

**Comment by lovelacecoding** [comment]
Alright. Then I'll use that instead. I'll come back to you :)

**Comment by lovelacecoding** [comment]
Okay so for the front page I made the following changes when it comes to contrast.

### The navbar
Made a slight change from #828282 to #737373. This passes the accessibility contrast check.

Before:
![image](https://user-images.githubusercontent.com/92865452/193428747-1555540e-69be-4d6d-a6e5-06b854172718.png)

After:
![image](https://user-images.githubusercontent.com/92865452/193428926-d8148a94-8e08-4b89-af77-055208c5f918.png)

### Get your walking buddy
Before:
![image](https://user-images.githubusercontent.com/92865452/193429075-0da70ea7-ee2b-4f22-9124-9eefbcd6c50d.png)

After:
![image](https://user-images.githubusercontent.com/92865452/193429095-71e2a00d-7234-4dff-8447-c0b956d64bf9.png)

### A proud partner
Before:
![image](https://user-images.githubusercontent.com/92865452/193429113-3d5496f3-5af8-47cd-98de-5dae1c3bb58d.png)
After:
![image](https://user-images.githubusercontent.com/92865452/193429130-ab7ce765-2915-4126-8aec-a628afb2ccaa.png)

### Your daily commute
Now this one is tricky, every highlight color is too light for a white background. So I decided that instead of a color to go for a bold font to not get away from the theme too much.

Before:
![image](https://user-images.githubusercontent.com/92865452/193429223-7a6a908f-30ca-41a6-a0c8-d5636b224ee3.png)

After:
![image](https://user-images.githubusercontent.com/92865452/193429249-07876464-e3d3-4781-94d9-3b16e60200b9.png)

Let me know what you like and do not like. All of these changes now pass the contrast check on [webaim](https://webaim.org/resources/contrastchecker/), which means it's accessible for most people and that search engines will be happier with you :)

**Comment by ashuvssut** [comment]
LGTM!

please make the PR

**Comment by lovelacecoding** [comment]
Done! You can find what I fixed on PR #40
